docs/yaml/yaml.go

```diff
@@ -185,11 +207,17 @@ func genFlagResult(flags *pflag.FlagSet) []cmdOption {
 		opt = cmdOption{
 			Option:       flag.Name,
 			ValueType:    flag.Value.Type(),
-			DefaultValue: forceMultiLine(flag.DefValue),
-			Description:  forceMultiLine(flag.Usage),
+			DefaultValue: forceMultiLine(flag.DefValue, 64 /* = 80 - '  default_value: ' */),
```

Comment:
nit: extract those constants in a `const` block

Reply:
added some consts